### PR TITLE
chore(Docker) Strip down Docker image size

### PR DESCRIPTION
**Pull request by igox**
Strip down Docker image size by a factor 12 (from 1.47GB to 120MB).
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,19 @@
-FROM node:lts-buster
+FROM node:lts-buster as build
```
Comment (scoulomb) [inline Dockerfile:1]: Thank you !
Nice to use multi-stage build :+1:

File: src/js/pdf-util.js [hunk 2]
```diff
@@ -96,8 +96,8 @@ export async function generatePdf (profile, reasons, pdfBase) {
   }
 
   drawText(profile.city, 105, 177, locationSize)
-  drawText(`${profile.datesortie}`, 91, 153, 11)
```
Comment (scoulomb) [inline src/js/pdf-util.js:99]: I would like to be iso with the official generation. What do you think?

File: src/js/pdf-util.js [hunk 3]
```diff
@@ -38,12 +38,12 @@ export async function generatePdf (profile, reasons, pdfBase) {
   } = profile
 
   const data = [
-    `Cree le: ${creationDate} a ${creationHour}`,
```
Comment (scoulomb) [inline src/js/pdf-util.js:41]: I would like to be iso with the official generation. What do you think?

**Review by scoulomb — COMMENTED** [review]
Thanks for the PR :)
I have some comments

**Comment by igox** [comment]
Closing this PR because based on `main` branch of my repo (!) and will recreate a proper one from a branch

**Pull request closed without merging**